pkg/reconciler/autoscaling/kpa/kpa.go

```diff
@@ -423,6 +434,37 @@ func (c *Reconciler) reconcileMetric(ctx context.Context, pa *pav1alpha1.PodAuto
 	return nil
 }
 
+func (c *Reconciler) checkContainerHealth(ctx context.Context, pa *pav1alpha1.PodAutoscaler) error {
```

Comment:
I expect `checkX` have no side effects. Yours actually changes PA status, so should be named differently.

Reply:
renamed to "updateContainerHealth".